app/code/Magento/Review/view/adminhtml/web/css/styles-custom.css

```diff
@@ -0,0 +1 @@
+.no-display-custom {display:none !important;}
```

Comment:
Is it possible to avoid the usage of `!important`?
It would be better to use the existent `.no-display` class whenever it's possible.